### ISSUE TITLE
Odd time units in update notification

### ISSUE THREAD

**Issue by thejacobtaylor** [minio/minio#4890]
Launch the server with an old version from the command line. The error message says: 
"You are running an older version of Minio released 3 months from now"
Expected
released 3 months ago

Example launch:
minio server /tmp/NodeA

┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ You are running an older version of Minio released 3 months from now ┃
┃ Update: https://dl.minio.io/server/minio/release/darwin-amd64/minio  ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used (`minio version`):
* Environment name and version (e.g. nginx 1.9.1):
* Server type and version:
* Operating System and version (`uname -a`):
* Link to your project:

**Comment by harshavardhana** [comment]
@thejacobtaylor this is actually fixed in the latest build. 

Fixed by 
```
commit cf1fc451427a56d2bc23015144d21fafb6438877
Author: Bala FA <bala.gluster@gmail.com>
Date:   Wed Apr 26 16:08:35 2017 +0530

    Improve duration humanization. (#4071)
```

and 
```
commit a0e02f43e1d02a6bac244cdae2839054be40d446
Author: Aditya Manthramurthy <donatello@users.noreply.github.com>
Date:   Wed May 31 09:22:00 2017 -0700

    Fix and cleanup update message and improve related tests (#4361)
    
    Fixes #4232
```

Newer message
```
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ You are running an older version of Minio released 12 months ago ┃
┃ Update: my_download_url                                          ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
```

**Comment by harshavardhana** [comment]
This is already fixed in current release @deekoder - this bug can be closed.
